### ISSUE TITLE
Until the first swipe the first slide doesn't show up

### ISSUE THREAD

**Issue by sahil290791** [leecade/react-native-swiper#416]
I have build a simple swiper in android where 5 slides are supposed to be shown. On load I can see 5 dots below with the first dot being active but the content is not shown for the same.

Here is my code:
```
<ScrollView>
        <Swiper
          height={deviceHeight - 200}
          paginationStyle={{ bottom: 20 }}
          activeDot={<View style={mainStyles.sliderBullets} />}
          loop={false}
        >
          <SliderContent
             // some content
          />
          <SliderContent
             // some content
          />
          <SliderContent
            // some content
          />
          <SliderContent
            // some content
          />
          <SliderContent
            // some content
          />
        </Swiper>
      </ScrollView>
```
If I swipe to second slide and come back to first one it shows the first slide.

Can anyone tell that why is the first slide empty until first swipe?

#### Additional Info: 
RN: 0.42.3
Android: 7.1.2 Nexus 5X
React native swiper: 1.5.4

**Comment by nixoz** [comment]
Here is the PR for this: https://github.com/leecade/react-native-swiper/pull/418

**Comment by jinxac** [comment]
+1, any solution to this @sahil290791  ?

**Comment by sahil290791** [comment]
@nixoz, I do not think this is ScrollViews issue, even If I remove `ScrollView`, the first swiper slides doesn't show up until first swipe.

**Comment by nixoz** [comment]
@sahil290791 it is.
1. Read the related issue https://github.com/facebook/react-native/issues/1831
2. I've patched swiper and tested it on mine app. so the patch is fixing the issue.

**Comment by sahil290791** [comment]
@nixoz that patch that you applied is only for `ios`, I am facing this problem in android.

**Comment by nixoz** [comment]
@sahil290791 - ahh, sorry man, my bad. Didn't paid attention to your's environment, coz faced that on ios ...ㅠㅠ

**Comment by sahil290791** [comment]
I guess [ViewPagerAndroid](https://facebook.github.io/react-native/docs/viewpagerandroid.html) is not playing well here.

**Comment by jinxac** [comment]
@nixoz it works with your PR on ios, but can anyone merge this request soon?

**Comment by lisa718** [comment]
@sahil290791 do you fix the problem on android?

**Comment by sahil290791** [comment]
@lisa718 Nope was unable, so I have enabled autoPlay now, which atleast doesn't show a blank slide.

**Comment by lisa718** [comment]
@sahil290791 OK，thanks

**Comment by douglaswissett** [comment]
Disabling removeClippedSubviews from the Swiper component fixed this issue for me.
```
              <Swiper
                height={190}
                showsButtons={true}
                removeClippedSubviews={false}>
```

**Comment by sahil290791** [comment]
Was this issue occurring in android for you?

**Comment by giautm** [comment]
@douglaswissett : Tks, it work. 👍

**Comment by lisa718** [comment]
when i push home button,and enter into app again,the image was gone.........

**Comment by lisa718** [comment]
and also at first time,the image could not show up.....

**Comment by wazggcd** [comment]
@douglaswissett cool! this issue bothered me for a long time. now it works!

**Comment by JayricMok** [comment]
@douglaswissett this facing this issue, I added `removeClippedSubviews={false}` to Swiper image also not showing

**Comment by JayricMok** [comment]
@lisa718 same as you. did you find any solution for this ?

**Comment by itanhduy** [comment]
@douglaswissett Thank you. It works.

**Comment by anotutoring** [comment]
@douglaswissett Thank you it wokr.
